cfpb/cfgov-refresh
# Cleanup related metadata

While looking at post preview I found some unused markup in related metadata.

## Removals

- `u-mb0` classes that appear to not be needed.
- text block paragraph that was not used.

## Changes

- Changes `half_width` boolean to `is_half_width`.

## Testing

1. Create a test document detail page in wagtail.
2. Add a related metadata to the sidebar.
3. Add the block types date, list, text, and topics.
4. Check the "is half width" option to check that that doesn't break.
5. The page appearance should be unchanged.

## Screenshots

Appearance should be unchanged:
![screen shot 2017-08-01 at 5 15 19 pm](https://user-images.githubusercontent.com/704760/28847388-0790fd82-76dd-11e7-8f0d-5122e7cf9681.png)
